packages/mask/src/UIRoot.tsx

```diff
@@ -39,6 +40,7 @@ function MaskThemeProvider({ children, baseline, useTheme }: MaskThemeProvider)
             </MaskIconPaletteContext.Provider>
         ),
         (jsx) => <ThemeProvider theme={theme} children={jsx} />,
+        (jsx) => <UIRuntimeProvider children={jsx} />,
```

Comment:
Maybe better as children of `<CustomSnackbar />`

Reply:
Indeed. Fixed.

Reply:
BTW, `<CustomSnackbar />` will be moved into `ui-runtime` in further refactoring.